### PR TITLE
Feat ISEEC 1777 refactoring modals

### PR DESCRIPTION
**Pull request by pironev**
## Context

Jira Issue:

Build Number:

## Checklist:

- [ ] Has been verified in an integrated environment
- [ ] Has relevant unit and integration tests passing
- [ ] Has no linting, test console, or browser console errors (best effort)
- [ ] Has JSDoc comment blocks for complex code

### CODE REVIEW

**Comment by pironev** [comment]
I modified Composed Modal that behaves as before and only if it put the "size" attribute (xs, sm, md, lg) will be added the carbon classes related to these dimensions.

File: src/components/ComposedModal/ComposedModal.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ ComposedModal.propTypes = {
   modalHeaderProps: PropTypes.object,
   modalTrigger: PropTypes.func,
   onCloseModal: PropTypes.func,
+  size: PropTypes.string,
```
Comment (Okisa) [inline src/components/ComposedModal/ComposedModal.js]: You can update this prop type to reflect the available options for size props.

```
PropTypes.oneOf([
    'xs',
    'sm',
    'md',
    'lg',
  ])
```

**Review by Okisa — COMMENTED** [review]
Looking good. I added a suggestion for the size props.

File: src/components/ComposedModal/ComposedModal.js (hunk 1 above)
Comment (pironev) [reply src/components/ComposedModal/ComposedModal.js]: done

**Review by LucasGrimauth — APPROVED** [review]
LGTM! 🥞

**Review by Okisa — APPROVED** [review]
Looking good!

**Pull request merged**